### DISCUSSION TITLE
Rule is not created  IPBan_Block_0

### DISCUSSION THREAD

**Discussion by Psalmir** [DigitalRuby/IPBan#153] [Q&A] [answered]
IPBAN service works
only two rules created in the firewall IPBan_EmergingThreats_0 and IPBan_EmergingThreats_1000
I tried to simulate brute-force from an external IP address - the IP address is not blocked after 5 incorrect attempts
the IPBan_Block_0 rule is not created in the firewall 
Help please
Windows Server 2012R2 x64
IPBan-Windows-x64_1_6_0

**Comment by jjxtra** [comment, marked as answer]
Turn off nla and ntlm. Please read the readme file.

**Reply by pro100it** [reply]
The rule is not created on the Windows 2016 server either (apparently because you need to create a rule with an IP address, at least 1.1.1.1) and until you restart the ip ban service. But this is wrong, ipban adds IP addresses to its database and to the firewall only after restarting the service. IP addresses are in the log, ip ban works, but adding real-time addresses to the firewall does not work.


Russian text:
Правило не создается и на виндовс 2016 сервер (видимо потому что надо создавать правило с ип адресом, хотя бы 1.1.1.1) и пока не перезапустишь службу ип бан. Но это неправильно ипбан добавляет в свою базу ип адреса а в файрвол только после перезапуска службы.  ИП адреса в журнале есть, ип бан работает но не работает добавление в реальном времени адресов в файервол

**Reply by jjxtra** [reply]
That's not the behavior I am observing on my servers. As soon as ips get blocked they show up in the firewall.

**Reply by pro100it** [reply]
Yep. Sorry all good/ The program works
